src/main/java/org/commcare/formplayer/beans/menus/BaseResponseBean.java

```diff
@@ -47,11 +50,15 @@ public void setNotification(NotificationMessage notification) {
         this.notification = notification;
     }
 
-    public boolean isClearSession() {
+    public void shouldAutoSubmit(boolean shouldAutoSubmit) {
+        this.shouldAutoSubmit = shouldAutoSubmit;
+    }
+
+    public boolean isclearSession() {
         return clearSession;
     }
 
-    public void setClearSession(boolean clearSession) {
+    public void setclearSession(boolean clearSession) {
```

Comment:
This make break the JSON output, which I believe is defined by these getters and setters